WordPress/src/main/java/org/wordpress/android/datasets/ReaderBlogTableWrapper.kt

```diff
@@ -11,4 +11,13 @@ class ReaderBlogTableWrapper
     fun isNotificationsEnabled(blogId: Long): Boolean = ReaderBlogTable.isNotificationsEnabled(blogId)
     fun setNotificationsEnabledByBlogId(blogId: Long, isEnabled: Boolean) =
             ReaderBlogTable.setNotificationsEnabledByBlogId(blogId, isEnabled)
+    fun isSiteFollowed(blogId: Long, feedId: Long?): Boolean {
+        return if (blogId != 0L) {
+            ReaderBlogTable.isFollowedBlog(blogId)
+        } else if (feedId != null && feedId != 0L) {
+            ReaderBlogTable.isFollowedFeed(feedId)
+        } else {
+            false
+        }
+    }
```

Comment:
Now after nowing that `blogId` can be indeed empty for an external feed I think this should be 
```suggestion
    fun isSiteFollowed(blogId: Long, feedId: Long?): Boolean {
        return if (feedId != null && feedId != 0L) {
            ReaderBlogTable.isFollowedFeed(feedId)
        } else if (blogId != 0L) {
            ReaderBlogTable.isFollowedBlog(blogId)
        } else {
            false
        }
    }
```

WDYT?

Reply:
Looks good to me.